### PR TITLE
Added ainb documentation and missing file extensions to the README

### PR DESCRIPTION
**Pull request by dt-12345**
I can also add my partial .asb documentation later - the ainb documentation is a direct conversion from my google doc with a few small edits

### CODE REVIEW

**Comment by dt-12345** [comment]
I do have some (very brief) notes for a few other file formats as well I could add

File: Readme.md [hunk 1]
```diff
@@ -8,134 +8,200 @@ File research for Tears of the Kingdom
...
 
 ## .bfsha
-
-```
-...
-```
+> **B**inary Ca**f**é **Sha**der
 
 ## .bgyml
+> **B**inary **G**enerated **Y**A**ML** (???)
```
Comment (ArchLeaders) [inline Readme.md:129]: Good guess, I like it.

**Review by ArchLeaders — APPROVED** [review]

**Comment by ArchLeaders** [comment]
> I do have some (very brief) notes for a few other file formats as well I could add

Go ahead, any bits of information are good.

Let me know when you're finished and I'll merge this.

**Comment by dt-12345** [comment]
That should be everything, I added a few hexpats (not all of them are necessarily complete) and some additional info on a few of the file formats.

The changes to BYML and BFEVFL should be the reason why many libraries for those two formats fail to parse a handful of files (e.g. VoiceSettingTable.byml or any file in romfs/Voice/BwavInfo)

**Pull request merged**